src/initialize.js

```diff
@@ -223,7 +252,7 @@ export async function initialize({
     publish(APP_PUBSUB_INITIALIZED);
 
     // Configuration
-    await handlers.config();
+    if (getConfig().MFE_CONFIG_API_URL) { await runtimeConfig(); } else { await handlers.config(); }
```

Comment:
I think I'd suggest moving this `if` statement logic _into_ the default config handler.  This logic here would continue to just say `await handlers.config();` and we would optionally call the `runtimeConfig` function if `MFE_CONFIG_API_URL` exists.  That way we enable folks to continue to override all of our default config handling.

Note that currently the `config` handler is set to `noOp`, so we'll just need to create a new one similar to `auth`, `analytics`, or `initError`.  Let me know if this doesn't make sense!

Reply:
I consider this, but many applications have the override handler to marge additional values for that specific MFE ([account example](https://github.com/openedx/frontend-app-account/blob/c39fd332b670f92879db3d227aaba83a1ad44e26/src/index.jsx#L60)), which make even if you have MFE_CONFIG_API_URL don't call the API.

That's why I put it there because you can set that additional values with the API or set by env file.

May be I could move the _if_ into `runtimeConfig` and call like: 
```javascript
    await handlers.config(); 
    await runtimeConfig(); 
```

What do you think?